crates/viewer/re_viewer_context/src/global_context/mod.rs

```diff
@@ -0,0 +1,50 @@
+//! Ultimately the goal is to have hierarchy of contexts from general to
+//! increasingly specific. Each part of the viewer should only have access a
+//! narrow context. Unfortunately, right now everything is still very
+//! intertwined with [`ViewerContext`](crate::ViewerContext) so we can't pull
+//! the [`GlobalContext`] out of this crate yet.
+
+mod app_options;
+mod command_sender;
+mod component_ui_registry;
+mod item;
+
+pub use app_options::AppOptions;
+
+use crate::ViewClassRegistry;
+pub use command_sender::{
+    command_channel, CommandReceiver, CommandSender, SystemCommand, SystemCommandSender,
+};
+pub use component_ui_registry::{ComponentUiRegistry, ComponentUiTypes};
+pub(crate) use item::resolve_mono_instance_path_item;
+pub use item::Item;
```

Comment:
nit: would be to have separate block for local use and exports (`pub use`) so it's easy to see at a glance what we're exporting from this module